app/services/idv/duplicate_ssn_finder.rb

```diff
@@ -10,7 +10,11 @@ def initialize(user:, ssn:)
     end
 
     def ssn_is_unique?
-      Profile.where.not(user_id: user.id).where(ssn_signature: ssn_signatures).empty?
+      associated_profiles_with_matching_ssn.empty?
+    end
+
+    def associated_profiles_with_matching_ssn
+      Profile.where.not(user_id: user.id).where(ssn_signature: ssn_signatures)
```

Comment:
Shouldn't we be checking _active_ profiles only?
Also, although the Postgres optimizer might be smart enough, it may be clearer if we write this as:
```
Profile.active.where(ssn_signature: ssn_signatures).where.not(user_id: user.id)
```

Reply:
Also, the other profiles should also have to be IAL2 with facial match.